### DISCUSSION TITLE
It seems that the element is lost in some cases (Version 4.4.2-7)

### DISCUSSION THREAD

**Discussion by smallrain1996** [neperfepx/neper#447] [Q&A] [answered]
When I ran the code below, I encountered confusion about element loss.
```
name=test
neper -T -n 2 -dim 2 -domain "square(350,200)" -morphooptiini "coo:file(coo),weight:0" -o $name
neper -M $name.tess -nset edges -faset none -rcl 0.5 -format inp,msh -elttype quad -o $name
neper -V $name.msh -datacellcol id -print $name
```
where, "coo" file is:
```
174.00 100.00 
176.00 100.00 
```

![image](https://user-images.githubusercontent.com/52028117/164390926-bcc5b5ee-ec82-4105-8f3d-bc6d8932d275.png)
[file.zip](https://github.com/rquey/neper/files/8528236/file.zip)

**Comment by rquey** [comment, marked as answer]
Converted to #448.

**Reply by rquey** [reply]
@smallrain1996 - this was a bug, now fixed. Thanks for reporting it.
